stdlib/scripts/build-stdlib.sh

```diff
@@ -27,7 +27,7 @@ if [ -z "${MOJO_OVERRIDE_COMPILER_VERSION_CHECK:-}" ]; then
     echo "Mismatch in compiler versions! Cannot build the standard library."
     echo "Expected compiler version: ${EXPECTED_COMPILER_VERSION}"
     echo "Current installed compiler version: ${ACTUAL_COMPILER_VERSION}"
-    echo "Please run modular update nightly/mojo to get the latest compiler."
+    echo "Please run 'magic update' and 'magic shell' to get the latest compiler."
```

Comment:
Maybe `magic update && magic install` to avoid having nested shells?

Reply:
Agreed. Fixed here: https://github.com/msaelices/mojo/commit/20251d817a2923f3e0aec514cc5187e182c13426